### DISCUSSION TITLE
Defining axes

### DISCUSSION THREAD

**Discussion by hosseinnegahban** [mdolab/dafoam#311] [General]
How can we determine the deformation axes in DAFoam 3? in version 2 we could do by shapey or shapex

**Comment by friedenhe** [comment]
Check https://github.com/mdolab/pygeo/blob/e36d6756cb855e6adc6337b5cd7c2c316b0c4c05/pygeo/mphys/mphys_dvgeo.py#L103

**Reply by hosseinnegahban** [reply]
Thank you very much for your reply. I found it, however, I want to define deformation both  in x and y direction with different bound. In previous version I only copied another line and changed it to shapex with different bound. What is the procedure in version 3?
